### PR TITLE
nodejs extension example

### PR DESCRIPTION
**Pull request by thewizarodofoz**
*Description of changes:*
Adding an extension example in Node.js

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by thewizarodofoz** [comment]
@julianwood @santiagocardenas would love to get this merged

File: nodejs-example-extension/nodejs-example-extension/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "secrets-extension",
```
Comment (santiagocardenas) [inline nodejs-example-extension/nodejs-example-extension/package.json]: Can you please change this to be more generic?

**Comment by santiagocardenas** [comment]
This looks great @thewizarodofoz !

Can you please confirm that you're contributing open source contibution MIT-0 and correct the nitpick on the package name?

Thanks!

**Comment by thewizarodofoz** [comment]
Thanks @santiagocardenas !
I updated the package name.
I hereby confirm that I am contributing open source contribution MIT-0.

**Comment by santiagocardenas** [comment]
Thank you for your contribution @thewizarodofoz !

**Pull request merged**